src/Symfony/Bridge/Monolog/Handler/NotifierHandler.php

```diff
@@ -57,13 +55,13 @@ public function handleBatch(array $records): void
     private function notify(array $records): void
     {
         $record = $this->getHighestRecord($records);
-        if (($record['context']['exception'] ?? null) instanceof \Throwable) {
-            $notification = Notification::fromThrowable($record['context']['exception']);
+        if (($record->context['exception'] ?? null) instanceof \Throwable) {
+            $notification = Notification::fromThrowable($record->context['exception']);
         } else {
-            $notification = new Notification($record['message']);
+            $notification = new Notification($record->message);
         }
 
-        $notification->importanceFromLogLevelName(Logger::getLevelName($record['level']));
+        $notification->importanceFromLogLevelName($record->level->getName());
```

Comment:
This looks to me like a psalm bug, if it cannot determine that a const array of strings is of type `string[]`.